jmh-core/src/main/java/org/openjdk/jmh/runner/Runner.java

```diff
@@ -200,22 +203,13 @@ public RunResult runSingle() throws RunnerException {
 
             if (lock == null) {
                 String msg = "Unable to acquire the JMH lock (" + JMH_LOCK_FILE + "): already taken by another JMH instance";
-                if (JMH_LOCK_IGNORE) {
-                    out.println("# WARNING: " + msg + ", ignored by user's request.");
-                } else {
-                    throw new RunnerException("ERROR: " + msg + ", exiting. Use -Djmh.ignoreLock=true to forcefully continue.");
-                }
+                throw new RunnerException("ERROR: " + msg + ", exiting. Use -Djmh.ignoreLock=true to forcefully continue.");
             }
 
             return internalRun();
         } catch (IOException e) {
             String msg = "Exception while trying to acquire the JMH lock (" + JMH_LOCK_FILE + "): " + e.getMessage();
-            if (JMH_LOCK_IGNORE) {
-                out.println("# WARNING: " + msg + ", ignored by user's request.");
-                return internalRun();
-            } else {
-                throw new RunnerException("ERROR: " + msg  + ", exiting. Use -Djmh.ignoreLock=true to forcefully continue.");
-            }
+            throw new RunnerException("ERROR: " + msg  + ", exiting. Use -Djmh.ignoreLock=true to forcefully continue.");
```

Comment:
Similarly, `throw new RunnerException("ERROR: Exception while trying to acquire the JMH lock (" + JMH_LOCK_FILE + "), exiting. Use -Djmh.ignoreLock=true to forcefully continue.", e);`

Reply:
Note the change above also links in the `IOException` as the cause.